examples/AspNetCore/README.md

```diff
@@ -0,0 +1,4 @@
+# OpenTelemetry ASP.NET Core Example Application
+
+This example ASP.NET Core Example Application is provided as a demo of how to
```

Comment:
`example` in `This example` looks redundant?

Reply:
Removed this file.